src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1442,6 +1446,129 @@ def disable_features(cmd, client, resource_group_name, cluster_name, features, k
+                print("You have an update pending. You can update the connectedk8s extension to latest v{} using 'az extension update -n connectedk8s'".format(latest_connectedk8s_version))
+
+        try:
+            # Fetch ConnectedCluster
+            connected_cluster = client.get(resource_group_name, cluster_name)
+            tr_logger.info("Connected cluster resource: {}".format(connected_cluster))
+        except Exception as ex:
+            not_found = False
+            try:
+                if ex.error.error.code == "NotFound" or ex.error.error.code == "ResourceNotFound":
+                    tr_logger.error("Connected cluster resource doesn't exist. " + str(ex))
+                    logger.error("The specified ConnectedCluster resource doesn't exist.")
+                    not_found = True
+            except Exception:
+                pass
+            if not not_found:
+                tr_logger.error("Couldn't check the existence of Connected cluster resource. Error: {}".format(str(ex)))
+
+        try:
+            # setting kubeconfig
+            kube_config = set_kube_config(kube_config)
+            # Loading the kubeconfig file in kubernetes client configuration
+            load_kube_config(kube_config, kube_context)
+            configuration = kube_client.Configuration()
+        except Exception as ex:
+            logger.error("Couldn't load kubeconfig. Error: {}".format(str(ex)))
+            tr_logger.error("Couldn't load kubeconfig. Error: {}".format(str(ex)))
+
+        # Install helm client
+        helm_client_location = install_helm_client()
+
+        try:
+            validate_release_namespace(client, cluster_name, resource_group_name, configuration, kube_config, kube_context, helm_client_location)
+        except Exception as e:
+            logger.error(str(e))
+            tr_logger.error(str(e))
+
+        kapi_instance = kube_client.CoreV1Api(kube_client.ApiClient(configuration))
+        try:
+            utils.try_list_node_fix()
+            api_response = kapi_instance.list_node()
+            required_node_counts = 0
+            for item in api_response.items:
+                node_arch = item.metadata.labels.get("kubernetes.io/arch")
+                node_os = item.metadata.labels.get("kubernetes.io/os")
+                if node_arch == "amd64" and node_os == "linux":
+                    tr_logger.info("Node {} has capacity: CPU: {}, and Memory: {} ".format(required_node_counts + 1, item.status.capacity.get("cpu"), item.status.capacity.get("memory")))
+                    required_node_counts += 1
+            if required_node_counts == 0:
+                tr_logger.error("There are no linux/amd64 nodes available on the kubernetes cluster.")
+                logger.error("This kubernetes cluster doesn't have any nodes with OS 'linux' and architecture 'amd64', for scheduling the Arc-Agents onto and connecting to Azure. Learn more at {}".format("https://aka.ms/ArcK8sSupportedOSArchitecture"))
```

Comment:
Can you clarify if this logger is just to the storage account or also to the CLI output? 

CLI should read:
****"The cluster does not have any nodes with****...rest is the same as existing string".****

Color: Red

Reply:
Yes it's the output to CLI as well. Whatever is coming with "logger" is the output to cli and whatever is coming with "tr_logger" is the output to the log file